src/frontend/src/lib/components/tokens/TokensSignedIn.svelte

```diff
@@ -24,12 +25,25 @@
 		({ id: tokenId }) =>
 			($balancesStore?.[tokenId]?.data ?? BigNumber.from(0n)).gt(0n) || displayZeroBalance
 	);
+
+	let animating = false;
+
+	const handleAnimationStart = () => (animating = true);
+
+	const handleAnimationEnd = () => (animating = false);
 </script>
 
 <TokensSkeletons>
 	{#each tokens as token (token.id)}
-		<Listener {token}>
-			<div in:fade>
+		<div
+			in:fade
+			out:fade
+			animate:flip={{ duration: 500 }}
```

Comment:
From watching the video, I would maybe speed up a bit the animation? Just thinking that overtime it can become boring.

Reply:
agreed